docker/Dockerfile.vanilla

```diff
@@ -9,5 +9,5 @@ USER firedrake
 WORKDIR /home/firedrake
 
 # Now install Firedrake.
-RUN curl -O https://raw.githubusercontent.com/firedrakeproject/firedrake/master/scripts/firedrake-install
-RUN bash -c "python3 firedrake-install --no-package-manager --disable-ssh"
+RUN curl -O https://raw.githubusercontent.com/firedrakeproject/firedrake/remove-build-files/scripts/firedrake-install
+RUN bash -c "python3 firedrake-install --no-package-manager --disable-ssh --remove-build-files"
```

Comment:
I presume we intend to leave this in here. Having small docker containers sounds like a Good Thing (and updating petsc in a docker container sounds odd).